### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in East Coast USA (New York City/Boston,etc)

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10343] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
https://github.com/apache/datafusion/discussions/10347

**Comment by matthewmturner** [comment]
I am definitely interested in a east coast meetup.  I am located in New York but go to Boston periodically so wouldnt have a problem going there.

**Reply by alamb** [reply]
Since I am based in Boston, it would be quite easy for me to attend / organize one here. NYC is easy for me to get too

**Comment by edmondop** [comment]
Definitely interested! I can help organize

**Comment by alamb** [comment]
Nice ! I think @philippemnoel [says he is interested on discord ](https://discord.com/channels/885562378132000778/1166447479609376850/1235688564247756871)as well 

How about a September 2024 timeframe? I could easily come down to NYC on the train. Maybe @pauldix  would want to come too

**Reply by philippemnoel** [reply]
I'm happy to help. We have an office at 22 Vanderbilt, which could easily host a group. It's immediately next to Grand Central, which would be perfect for anyone coming from Boston. There are pretty large meeting rooms + nice common areas.

**Reply by alamb** [reply]
Sounds like we should make a save the date and we'll have ourselves a NYC meetup

**Reply by pauldix** [reply]
I will definitely come if there's an NYC meetup!

**Reply by MrPowers** [reply]
I would be happy to join a NYC meetup as well!

**Reply by alamb** [reply]
We are going to hold one https://github.com/apache/datafusion/discussions/11213 -- Sep 17, 2024

More info coming soon
